traits/trait_list_object.py

```diff
@@ -52,6 +52,12 @@ def __init__(self, index=0, removed=None, added=None):
             added = []
         self.added = added
 
+    def __repr__(self):
+        outstr = "TraitListEvent(index={}, removed={}, added={})".format(
```

Comment:
It's partly just a matter of personal style, but it's fine to simply `return` the constructed string directly here (without first binding it to a name), and similarly elsewhere. Up to you whether you want to make that change or not - it's also fine as it is.